### PR TITLE
v1.10: ofi mtl: also link in mtl_ofi_LIBS in the static case

### PR DESCRIPTION
**Pull request by jsquyres**
(cherry picked from commit open-mpi/ompi@1e9227765ac86afbd9d15faf43b72976e29c64f7)

Found by @PHHargrove http://www.open-mpi.org/community/lists/devel/2015/08/17752.php

### CODE REVIEW

**Comment by jsquyres** [comment]
@rhc54 or @yburette please review

**Comment by rhc54** [comment]
Looks right to me - thx!

:+1:

**Comment by yburette** [comment]
I'm still getting errors when configuring with
`--enable-static --disable-shared --with-libfabric=yes`

Are you guys able to build it this way?
What am I missing?

**Comment by jsquyres** [comment]
What errors are you getting?

**Comment by jsquyres** [comment]
FWIW, I was actually configuring with `--enable-static --disable-shared --with-libfabric=/path/to/libfabric/install`

**Comment by yburette** [comment]
Tried again and I get these errors:

```
mtl_ofi_component.c:(.text+0x306): undefined reference to `fi_dupinfo'
mtl_ofi_component.c:(.text+0x360): undefined reference to `fi_getinfo'
mtl_ofi_component.c:(.text+0x371): undefined reference to `fi_strerror'
mtl_ofi_component.c:(.text+0x3a5): undefined reference to `fi_freeinfo'
mtl_ofi_component.c:(.text+0x3b2): undefined reference to `fi_freeinfo'
mtl_ofi_component.c:(.text+0x56a): undefined reference to `fi_fabric'
mtl_ofi_component.c:(.text+0x59b): undefined reference to `fi_strerror'
mtl_ofi_component.c:(.text+0x66d): undefined reference to `fi_strerror'
mtl_ofi_component.c:(.text+0x7ef): undefined reference to `fi_strerror'
mtl_ofi_component.c:(.text+0x822): undefined reference to `fi_strerror'
mtl_ofi_component.c:(.text+0x883): undefined reference to `fi_strerror'
../../../ompi/.libs/libmpi.a(mtl_ofi_component.o):mtl_ofi_component.c:(.text+0x8d4): more undefined references to `fi_strerror' follow
../../../ompi/.libs/libmpi.a(mtl_ofi_component.o): In function `ompi_mtl_ofi_component_init':
mtl_ofi_component.c:(.text+0x9a0): undefined reference to `fi_freeinfo'
mtl_ofi_component.c:(.text+0x9aa): undefined reference to `fi_freeinfo'
mtl_ofi_component.c:(.text+0x9e7): undefined reference to `fi_strerror'
../../../ompi/.libs/libmpi.a(mtl_ofi.o): In function `ompi_mtl_ofi_del_procs':
mtl_ofi.c:(.text+0x1a2a): undefined reference to `fi_strerror'
collect2: error: ld returned 1 exit status
make[2]: *** [ompi_info] Error 1
```

**Comment by yburette** [comment]
Hold on, I think I know what's going on... User error. Let me try again.

**Comment by yburette** [comment]
OK, sorry for my clumsiness. All is good :+1:

**Comment by jsquyres** [comment]
Whew.  I was getting worried there... :smile:
